### DISCUSSION TITLE
Add jumplist support

### DISCUSSION THREAD

**Discussion by devanshuDesai** [qutebrowser/qutebrowser#6113] [Ideas] [answered]
I think it would be pretty useful to have support for commands like `<C-o>` and `<C-i>` as well as maybe support for marks would be mighty useful for browsing documentation.

**Comment by The-Compiler** [comment, marked as answer]
Marks are already implemented, see e.g. ``` `a ``` and `'a` (`ma` is already used for quickmarks) - as well as `''` to jump back after e.g. searching.

As for jumplist commands, see ~~#349~~ #2642.

**Reply by The-Compiler** [reply]
Whoops, I meant #2642 of course...
